entries/fadeIn.xml

```diff
@@ -40,7 +40,7 @@ $( "#clickme" ).click(function() {
       <img class="column three" src="/resources/0042_06_34.png" alt=""/>
       <img class="column three" src="/resources/0042_06_35.png" alt=""/>
       <img class="column three" src="/resources/0042_06_36.png" alt=""/>
-      <figcaption>figure 1</figcaption>
+      <figcaption>Figure 1 - Illustration of the <code>fadeIn()</code> effect</figcaption>
```

Comment:
After an additional check, I think the case doesn't apply to this example. So, I'll leave it as per @agcolom commit.